testsuite/src/it/java/org/graalvm/tests/integration/utils/versions/MandrelVersionCondition.java

```diff
@@ -42,6 +43,8 @@ public class MandrelVersionCondition implements ExecutionCondition {
     private static final ConditionEvaluationResult ENABLED_BY_DEFAULT =
             enabled("@IfMandrelVersion is not present");
 
+    private static final Pattern jdkPattern = Pattern.compile("(?<jfeature>[0-9]+)(\\.(?<jinterim>[0-9]*)\\.(?<jupdate>[0-9]*))?");
```

Comment:
Nitpick: consider renaming field:

```suggestion
    private static final Pattern JDK_PATTERN = Pattern.compile("(?<jfeature>[0-9]+)(\\.(?<jinterim>[0-9]*)\\.(?<jupdate>[0-9]*))?");
```